src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -41,6 +56,24 @@
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(WebACAuthorizationDelegate.class);
 
+    private static final Map<String, String> actionMap;
+
+    static {
+        final Map<String, String> map = new HashMap<>();
+        // WEBAC_MODE_READ Permissions
+        map.put(READ, WEBAC_MODE_READ_VALUE);
+        // WEBAC_MODE_WRITE Permissions
```

Comment:
I don't believe there exist `DELETE` or `UPDATE` actions (but c.f. `REMOVE` and `SET_PROPERTY`). The full list of MODE permissions are here: http://git.io/vZUuD (github) and http://docs.jboss.org/modeshape/4.2.0.Final/api/org/modeshape/jcr/ModeShapePermissions.html (javadocs)